client/scripts/DrawTogether.js

```diff
@@ -638,7 +649,17 @@ DrawTogether.prototype.joinGame = function joinGame () {
 			this.chat.addMessage("Welcome to anondraw, enjoy your game!");
 			this.chat.addMessage("Invite friends:", "http://www.anondraw.com/#" + room);
 			
-			this.getFavorites();
```

Comment:
This is not a good way to do it. Randomly waiting gives very very very very very big problems down the line.
If you want to do something when the server knows our rep; just put it in the event handler where we get our reputation.